pages/idea/new.tsx

```diff
@@ -210,6 +210,7 @@ const CreateIdeaPage = ({ community }: { community: Community }) => {
       label: "Nouns DAO Prop",
       value: TagType.Nouns,
       requiredTokens: 8,
+      disabled: community?.uname !== SUPPORTED_SUBDOMAINS.LIL_NOUNS
```

Comment:
Removing the nouns tag from nouns domain as i don't think it's relevant for nouns.